packages/react/src/domain-config/ConfigView.tsx

```diff
@@ -79,34 +105,42 @@ function ConfigView() {
     };
 
     const submitDm3UsernameClaim = async () => {
-        const signedProfile = await Lib.account.getUserProfile(
-            state.connection,
-            state.connection.account!.ensName,
-        );
+        setLoading('dm3UserName');
+        setError(undefined);
+        try {
+            const signedProfile = await Lib.account.getUserProfile(
+                state.connection,
+                state.connection.account!.ensName,
+            );
 
-        await Lib.external.claimSubdomain(
-            state.connection.account!,
-            'http://localhost:8081/profile',
-            dm3UserEnsName! + Lib.GlobalConf.USER_ENS_SUBDOMAIN(),
-            signedProfile!,
-        );
+            await Lib.external.claimSubdomain(
+                state.connection.account!,
+                'http://localhost:8081/profile',
+                dm3UserEnsName! + Lib.GlobalConf.USER_ENS_SUBDOMAIN(),
+                signedProfile!,
+            );
 
-        await Lib.external.createAlias(
-            state.connection.account!,
-            state.connection,
-            state.connection.account!.ensName,
-            dm3UserEnsName! + Lib.GlobalConf.USER_ENS_SUBDOMAIN(),
-            state.auth.currentSession!.token!,
-        );
+            await Lib.external.createAlias(
+                state.connection.account!,
+                state.connection,
+                state.connection.account!.ensName,
+                dm3UserEnsName! + Lib.GlobalConf.USER_ENS_SUBDOMAIN(),
+                state.auth.currentSession!.token!,
+            );
 
-        dispatch({
-            type: ConnectionType.ChangeAccount,
-            payload: {
-                ...state.connection.account!,
-                ensName: dm3UserEnsName + Lib.GlobalConf.USER_ENS_SUBDOMAIN(),
-            },
-        });
-        window.location.reload();
+            dispatch({
+                type: ConnectionType.ChangeAccount,
+                payload: {
+                    ...state.connection.account!,
+                    ensName:
+                        dm3UserEnsName + Lib.GlobalConf.USER_ENS_SUBDOMAIN(),
+                },
+            });
+            window.location.reload();
```

Comment:
Why do we have to reload here?

Reply:
This should be removed in a separate issue. For now, it was easier to re-sign-in instead of making all the changes in the background